dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/AbstractJdbcTableManager.java

```diff
@@ -585,6 +587,19 @@ protected List<AnalyticsTableColumn> addOrganisationUnitLevels()
             .collect( Collectors.toList() );
     }
 
+    /**
+     * Organisation unit name hierarchy delivery.
+     *
+     * @return a table column {@link AnalyticsTableColumn}
+     */
+    protected AnalyticsTableColumn getOrganisationUnitNameHierarchyColumn()
+    {
+        String columnAlias = "concat_ws('/'," + organisationUnitService.getFilledOrganisationUnitLevels().stream()
```

Comment:
I think Jan wanted a spaced character. Instead of `/`, I believe he expects ` / `. This is the format described in the Jira DHIS2-14683. Better double-check with him.